test/base-commands/twilio-client-command.test.js

```diff
@@ -230,5 +234,69 @@ describe('base-commands', () => {
         expect(ctx.stderr).to.contain('A fake API error');
       });
     });
+
+    describe('regional and edge support', () => {
+      const envTest = (
+        args = [],
+        { envRegion = undefined, envEdge = undefined, configEdge = undefined } = {}
+      ) => {
+        return test
+          .do(ctx => {
+            ctx.userConfig = new ConfigData();
+            if (configEdge) {
+              ctx.userConfig.edge = configEdge;
+            }
+            if (envRegion) {
+              process.env.TWILIO_REGION = envRegion;
+            }
+            if (envEdge) {
```

Comment:
These `if` checks aren't necessary.

Reply:
Yeah they are. If I don't check, it sets process.env.TWILIO_WHATEVER to the string `undefined`.

Reply:
But that's fine, right?

Reply:
Well, why would it be a string?

Reply:
I'm not sure. I haven't been able to figure that one out